packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -98,6 +98,70 @@ enum DragAnchor {
 /// widget always displays [child].
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=QzA4c4QHZCY}
+/// 
+/// {@tool dartpad --template=stateful_widget_material}
+///
+/// The following example has a [Draggable] widget along with a [DragTarget] 
+/// in a row demonstrating an incremented `acceptedData` integer value when 
+/// you drag the element to the target.
+///
+/// ```dart
+/// var acceptedData = 0;
+/// 
+/// Widget build(BuildContext context) {
+///   return Row(
+///     mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+///     children: [
+///       Draggable(
+///         // Data is the value this Draggable stores.
+///         data: 10,
+///         child: Container(
+///           height: 100.0,
+///           width: 100.0,
+///           color: Colors.lightGreenAccent,
+///           child: Center(
+///             child: Text("The Draggable")
+///           ),
+///         ),
+///         feedback: Container(
+///           height: 100.0,
+///           width: 100.0,
+///           color: Colors.deepOrange,
+///           child: Icon(Icons.directions_run)
+///         ),
+///         childWhenDragging: Container(
+///           height: 100.0,
+///           width: 100.0,
+///           color: Colors.pinkAccent,
+///           child: Center(
+///             child: Text("Child When Dragging")
+///           ),
+///         ),
+///       ),
+///       DragTarget(
+///         builder: (BuildContext context) {
+///             Container(
+///               height: 100.0,
+///               width: 100.0,
+///               color: Colors.cyan,
+///               child: Center(
+///                 child: Text("Value is updated to: $acceptedData"),
+///               ),
+///             ),
+///          }
+///          onAccept: (int data){
+///            setState((){
+///             acceptedData += data;
+///            });
+///          }
+///       )
+///     ]
```

Comment:
```suggestion
///          },
///       ),
///     ],
```

Reply:
It also seems like the indentation here is a little off -- there's 3 spaces instead of 2 between this line and the next.